nats-top.yaml

```diff
@@ -0,0 +1,57 @@
+package:
+  name: nats-top
+  version: 0.6.3
+  epoch: 0
+  description: A top-like tool for monitoring NATS servers.
+  copyright:
+    - license: MIT
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/nats-io/nats-top
+      tag: v${{package.version}}
+      expected-commit: 812d35cc991232d7606816a15b2ea0b12fb129e7
+
+  - uses: go/build
+    with:
+      packages: ./nats-top.go
+      output: nats-top
+      extra-args: -cover
```

Comment:
@kranurag7 I don't think we want to publish the main package with the instrumented binary, but having it as as subpackage, because it has binary size and performance consequnces